eng/tools/typespec-validation/src/rules/sdk-tspconfig-validation.ts

```diff
@@ -128,6 +128,77 @@ class TspconfigEmitterOptionsSubRuleBase extends TspconfigSubRuleBase {
     return option;
   }
 
+  protected resolveVariables(value: string, config: any): { resolved: string; error?: string } {
```

Comment:
Here is to resolve the issue https://github.com/Azure/azure-rest-api-specs/issues/38964.